Task_DEV_6/Task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace Task_DEV_6
+{
+    class CustomStack
+    {
+        private const int LENGTH = 10;
+        private const string EXCEPTION_MESSAGE = "There are no elements on the stack.";
+
+        private int[] elements;
+        private int count;
+
+        public CustomStack()
+        {
+            elements = new int[LENGTH];
+        }
+
+        public CustomStack(int capacity)
+        {
+            elements = new int[capacity];
+        }
+
+        public int Capacity()
+        {
+            return elements.Length;
+        }
+
+        public int Count()
+        {
+            return count;
+        }
+
+        public bool isEmpty()
+        {
+            if (count == 0)
+            {
+                return true;
+            }
+            return false;
+        }
+
+        public int Pop()
+        {
+            ThrowExeption();
+            if (count == elements.Length / 2.0)
+            {
+                int size = elements.Length / 2;
+                int[] newNumbers = new int[size];
+                for (int i = 0; i < elements.Length; i++)
+                {
+                    newNumbers[i] = elements[i];
+                }
+                elements = newNumbers;
+            }
```

Comment:
You have the same code in `Push` method. Why don't you move this logic in separate method `Resize`?

Reply:
Done.